src/main/java/sirius/biz/storage/layer3/VirtualFileSystem.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.storage.layer3;
+
+import sirius.kernel.commons.Strings;
+import sirius.kernel.di.std.PriorityParts;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.health.Exceptions;
+
+import javax.annotation.Nonnull;
+import java.util.List;
+import java.util.Optional;
+
+/**
+ * Provides the main entrypoint into the <b>Virtual File System</b>.
+ * <p>
+ * This serves mainly three purposes:
+ * <ol>
+ * <li>Making internally stored files accessible to the outside by using <b>downlinks</b> like FTP, SFTP or SCP</li>
+ * <li>Making external files accessible to the software by using <b>uplinks</b> like FS, CIFS, SFTP</li>
+ * <li>Provide a uniform management UI for all file based operations within sirius</li>
+ * </ol>
+ */
+@Register(classes = VirtualFileSystem.class)
```

Comment:
No framework annotation?

Reply:
does no harm - therefore there is no need to disable...